delta_poker/marketing_main.py

```diff
@@ -6,9 +9,11 @@
 from sb3_contrib.common.wrappers import ActionMasker
 from torch import nn
 
-from game_mechanics import HERE, PokerEnv, human_player, wait_for_click
+from game_mechanics import HERE, PokerEnv, choose_move_randomly, human_player, wait_for_click
 
-NEURAL_NETWORK = MaskablePPO.load(HERE / "saved_network")
+NEURAL_NETWORK = MaskablePPO.load(
+    "/Users/jamesrowland/Code/poker/delta_poker/nolimit_checkpoint1.zip"
```

Comment:
Think this whole file shouldn't be here...

Reply:
Perhaps it should be on a specific branch? Even then, a smart student could find it...

Reply:
mmmm we can take it our if you like, why do you think it's bad if students see it?

Reply:
Only thing I was thinking was that it might look bad to students that we use stable baselines, but you're right - that actually doesn't matter.

I now object on another ground - this isn't part of the poker game. Imo this should just live as a replit and not on the poker github. It uses packages not in `requirements.txt` and isn't necessary to run the game.

What do you think?